src/components/avatar/Avatar.tsx

```diff
@@ -151,10 +154,10 @@ const Avatar = ({
     className,
   );
 
-  const enableTooltip = tooltip && typeof fullName === 'string' && fullName.length > 0;
+  const enableTooltip = tooltip && ((typeof fullName === 'string' && fullName.length > 0) || tooltipProps);
 
   const Component = enableTooltip ? TooltippedBox : Box;
-  const tooltipProps = enableTooltip
+  const defaultToolTipProps = enableTooltip
     ? {
         tooltip: <TextBodyCompact>{fullName}</TextBodyCompact>,
         tooltipColor: 'white',
```

Comment:
Maybe add these to the tooltipProps too so it's easier to make it look consistent
```
{
  tooltipColor: 'white',
  tooltipPosition: 'top',
  tooltipSize: 'small',
  ...tooltipProps
}
```